### ISSUE TITLE
[Feature][release] New tool find out not cherry-pick pr with specific milestone

### ISSUE THREAD

**Issue by zhongjiajie** [apache/dolphinscheduler#11289]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

when we release minor version of dolphinscheudler, we will add milestone to issue or PR, and cherry-pick them when PR merged, currently we have to find not cherry-pick PR by manual, maybe we should add a tool to make it easier

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by zhongjiajie** [comment]
I do some research to https://cli.github.com , it work well in PR by

```bash
gh pr list --state merged --search "milestone:3.0.0-release" --json number,title --template '{{range .}}{{tablerow (printf "#%v" .number | autocolor "green") .title}}{{end}}' --limit 10000
```

but when we try to find out some issue with milestone and the linked PR without milestone, it is do not work expect, only can filter issue

```bash
gh issue list --state closed --milestone "3.0.0-release" --search "-label:wontfix" --json number,title,url --template '{{range .}}{{tablerow ( .number | autocolor "green") .title .url}}{{end}}' --limit 10000
```

**Comment by zhongjiajie** [comment]
I will add some bash or python script to do that
